src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -87,9 +87,9 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
             attach_disk_command = 'az vm disk attach -g {g} --vm-name {repair} --name {id}' \
                                   .format(g=repair_group_name, repair=repair_vm_name, id=copy_disk_id)
 
-            logger.info('Creating repair vm...')
+            logger.info('Creating repair VM...')
             _call_az_command(create_repair_vm_command, secure_params=[repair_password])
```

Comment:
Can we add logic to `_call_az_command` to assert that the command passed in starts with "az "?